editor/src/components/editor/project-owner-hooks.ts

```diff
@@ -21,33 +20,25 @@ export function useDisplayOwnershipWarning(): void {
       // Without this check this hook will only fire before the LOAD action
       // and then the toasts will be cleared before they ever really existed.
       if (ownershipValues.projectID != null) {
-        switch (ownershipValues.projectOwnership) {
-          case 'yes':
-            // Remove the toast if we switch to a project that the user owns.
-            globalThis.requestAnimationFrame(() => {
-              dispatch([removeToast(OwnershipToastID)])
-            })
-            break
-          case 'no':
-            // Add the toast if we switch to a project that the user does not own.
-            globalThis.requestAnimationFrame(() => {
-              dispatch([
-                showToast(
-                  notice(
-                    'Viewer Mode: As you are not the owner of this project, it is read-only.',
-                    'NOTICE',
-                    true,
-                    OwnershipToastID,
-                  ),
+        if (ownershipValues.allowedToEdit) {
+          // Remove the toast if we switch to a project that the user owns.
+          globalThis.requestAnimationFrame(() => {
+            dispatch([removeToast(OwnershipToastID)])
+          })
+        } else {
+          // Add the toast if we switch to a project that the user does not own.
+          globalThis.requestAnimationFrame(() => {
+            dispatch([
+              showToast(
+                notice(
+                  'Viewer Mode: As you are not the owner of this project, it is read-only.',
```

Comment:
I think this warning text is now confusing, as we'll be showing this even if you are the owner of the project but have it open in another tab. If we can detect at this point that you're the owner but still in viewer mode then we should adjust the message to reflect that. If not we should either remove the part about "you are not to the owner" or include something about it maybe being open in another tab

Reply:
That's a good shout.

Reply:
Fixed in fcc60802d.